internal/config/plugin.go

```diff
@@ -17,6 +17,8 @@ type Plugin struct {
 		Registry bool `hcl:"registry,optional"`
 		Platform bool `hcl:"deploy,optional"`
 		Releaser bool `hcl:"release,optional"`
+		Pipeline bool `hcl:"pipeline,optional"`
+		Step     bool `hcl:"step,optional"`
```

Comment:
These should not be here, we're not doing any plugin types yet and these aren't what they will be called.